web/src/app/monitor/components/task-report/task-report.component.html

```diff
@@ -3,7 +3,7 @@
       <h4>{{reportData.meta.name}} ({{reportData.meta.date | dateTimeZone}})</h4>
   </div>
   <div class="panel-body">
-    <div style="display: block" class =" col-xs-12 col-md-7 col-md-offset-2">
+    <div style="display: block" class ="col-xs-12 col-md-7 col-md-offset-2 monitor-chart">
```

Comment:
Is this class needed? I can't see a usage.
In general, we are using BEM class convention http://getbem.com/naming/ and the class name should start with the name of the component. So `monitor-chart` should be `task-report__monitor-chart` in case you need the class.

Reply:
Removed